src/Pact/Types/Term.hs

```diff
@@ -1253,7 +1266,41 @@ termEq (TSchema a b c d _) (TSchema e f g h _) = a == e && b == f && c == g && a
 termEq (TVar a _) (TVar b _) = a == b
 termEq _ _ = False
 
+instance Eq1 Guard where liftEq = $(makeLiftEq ''Guard)
```

Comment:
This one I'm less concerned about as these things do have a tendency to break, so forward-looking changes are 👍